src/ui/buttons/props.js

```diff
@@ -841,5 +925,6 @@ export function normalizeButtonProps(
     supportedNativeBrowser,
     showPayLabel,
     displayOnly,
+    message,
```

Comment:
```suggestion
    message: message ? normalizeButtonMessage(props, message) : undefined,
```

and then remove the `if (message)` statement. I think its better if we don't override params/options from objects. you'll find it all over the place in this codebase but it makes it harder to understand when values are changing